### PR TITLE
chore: update README.md

### PR DESCRIPTION
**Pull request by Anto79-ops**
to obtain keys

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,17 @@ Platform | Description
 `binary_sensor` | On/Off sensors for various settings.
 `sensor` | Show info from an Renogy's API.
 
+**Obtaining your Renogy API Keys
```
Comment (firstof9) [inline README.md:23]:
```suggestion
## Obtaining your Renogy API Keys
```

**Review by firstof9 — CHANGES_REQUESTED** [review]
These changes will make the text larger like the text for "Installation via HACS"

**Review by Anto79-ops — COMMENTED** [review]
Spelling

**Review by firstof9 — APPROVED** [review]
Looks good.

**Comment by firstof9** [comment]
Thanks!

**Pull request merged**